docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -46,11 +54,16 @@ Step 1: Prerequisites
 
 * You will need the public keys of all the existing BigchainDB nodes.
 
+* Client Certificate for the new BigchainDB Server to identify itself to MongoDB Primary instance.
+
 * A new Kubernetes cluster setup with kubectl configured to access it.
 
 * Some familiarity with deploying a BigchainDB node on Kubernetes.
   See our :doc:`other docs about that <node-on-kubernetes>`.
 
+* *If* you want to deploy monitoring and backup agents as well, you will need a certificate for MongoDB monitoring
```

Comment:
We will need multiple monitoring and backup agents too as the Cloud Manager selects a master among them and that instance does the monitoring or backup accordingly.
So, multiple monitoring and backup agents' deployment should be essential, rather than optional.